src/jobs/microk8s_install.yml

```diff
@@ -124,24 +130,88 @@ steps:
         END_VALUES
 
   - run:
-      name: Get chart and setup codacy helm repos
+      name: Setup helm repos
       command: |
         set -x
-        git clone https://github.com/codacy/chart.git
-        cd chart/
-        git checkout << parameters.chart_branch >>
-
         sudo helm repo add codacy-stable    https://charts.codacy.com/stable
```

Comment:
I'm just adding all the repos because your chart might depend on components that are in different repos. For instance, all the components from stable except for the component you are working on, which is in unstable. But I'm inclined to move this to a parameter, which also makes the it more general.

Reply:
> But I'm inclined to move this to a parameter, which also makes the it more general.

yes please